modules/aviatrix-controller-initialize/variables.tf

```diff
@@ -0,0 +1,74 @@
+data "aws_caller_identity" "current" {}
+
+data "aws_region" "current" {}
+
+variable "controller_launch_wait_time" {
+  type        = number
+  description = "Controller launch wait time"
+  default     = 210
+}
+
+variable "aws_account_id" {
+  type        = string
+  description = "aws account id"
+}
+
+variable "public_ip" {
+  type        = string
+  description = "aviatrix controller public ip address(required)"
+}
+
+variable "private_ip" {
+  type        = string
+  description = "aviatrix controller private ip address(required)"
+}
+
+variable "admin_email" {
+  type        = string
+  description = "aviatrix controller admin email address"
+}
+
+variable "admin_password" {
+  type        = string
+  description = "aviatrix controller admin password"
+}
+
+variable "access_account_email" {
+  type        = string
+  description = "aviatrix controller access account email"
+}
+
+variable "controller_version" {
+  type        = string
+  default     = "latest"
+  description = "The version in which you want launch Aviatrix controller"
+}
+
+variable "access_account_name" {
+  type        = string
+  description = "Access account name"
+}
+
+variable "customer_license_id" {
+  type        = string
+  description = "aviatrix customer license id"
+  default     = ""
+}
+
+variable "ec2_role_name" {
+  type        = string
+  description = "EC2 role name"
+  default     = "aviatrix-role-ec2"
+}
+
+variable "app_role_name" {
+  type        = string
+  description = "APP role name"
+  default     = "aviatrix-role-app"
+}
+
+locals {
+  ec2_role_name       = var.ec2_role_name != "aviatrix-role-ec2" ? var.ec2_role_name : "aviatrix-role-ec2"
```

Comment:
I made this change to `ec2_role_name` and `app_role_name` simply from a documentation point of view. If we say "if this is not set, aviatrix-role-ec2 will be used", I think it's better to set aviatrix-role-ec2 as the default value, instead of hiding it in the code.